osu.Game.Rulesets.Osu/Mods/OsuModZoomed.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Localisation;
+using osu.Framework.Utils;
+using osu.Game.Configuration;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Overlays.Settings;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Rulesets.UI;
+using osu.Game.Scoring;
+using osuTK;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    internal class OsuModZoomed : Mod, IUpdatableByPlayfield, IApplicableToScoreProcessor
+    {
+        public override string Name => "Zoomed";
+        public override string Acronym => "ZM";
+        public override IconUsage? Icon => FontAwesome.Solid.Glasses;
+        public override ModType Type => ModType.Fun;
+        public override LocalisableString Description => "Big brother is watching your cursor.";
+        public override double ScoreMultiplier => 1;
+        public override Type[] IncompatibleMods => new[] { typeof(OsuModBarrelRoll) };
+
+        [SettingSource("Delay", "Delay in milliseconds for the view to catch up to the cursor")]
+        public BindableInt MovementDelay { get; } = new BindableInt
+        {
+            MinValue = 0,
+            MaxValue = 1000,
+            Precision = 100,
+        };
+
+        [SettingSource("Initial zoom", "The starting zoom level")]
+        public BindableDouble InitialZoom { get; } = new BindableDouble(1.5)
+        {
+            MinValue = 1,
+            MaxValue = 2,
+            Precision = 0.05
+        };
+
+        [SettingSource("Final zoom", "The starting zoom level")]
+        public BindableDouble FinalZoom { get; } = new BindableDouble(1.8)
+        {
+            MinValue = 1,
+            MaxValue = 2,
+            Precision = 0.05
+        };
+
+        [SettingSource("Final zoom at combo", "The combo count at which point the zoom level stops increasing.", SettingControlType = typeof(SettingsSlider<int, OsuSliderBar<int>>))]
+        public BindableInt FinalZoomCombo { get; } = new BindableInt
+        {
+            Default = 100,
+            Value = 100,
+            MinValue = 0,
```

Comment:
this should be 1 , 0 causes a crash ( not sure why, i assume it might be because it has yet to be updated with any combo information and is left with the combo zoomed value and the default value together? ) , and imo makes little sense regardless since if max zoom is being achieved at 0 combo then the setting the final and initial value to the same zoom does the job just fine

Reply:
Fixed.